lib/yam/agama/agama_base.pm

```diff
@@ -33,4 +33,8 @@ sub post_run_hook {
     ensure_serialdev_permissions;
 }
 
+sub test_flags {
+    return {fatal => 1, milestone => 1};
```

Comment:
I think we don't need to set milestone since we haven't such requirement for a snapshot here.

Reply:
I just use this flag to stop failed case, not used for make snapshot here.

Reply:
If just want to stop when failed, we can just set 'fatal=1', 'milestone' is for a snapshot for the current test environment.

Reply:
afir milestone shouldn't be needed. seems that PR need rebase now.

Reply:
@jknphy PR arleady rebase and VR done.